instrumentation/log4j/log4j-appender-2.16/library/README.md

```diff
@@ -0,0 +1,74 @@
+# Log4j2 Appender
+
+This module provides a Log4j2 [appender](https://logging.apache.org/log4j/2.x/manual/appenders.html)
+which forwards Log4j2 log events to
+the [OpenTelemetry Log SDK](https://github.com/open-telemetry/opentelemetry-java/tree/main/sdk/logs)
+.
+
+To use it, add the following modules to your application's classpath.
+
+**Maven**
+
+```xml
+<dependencies>
+  <dependency>
+    <groupId>io.opentelemetry.instrumentation</groupId>
+    <artifactId>opentelemetry-log4j-appender-2.16</artifactId>
+    <version>{version}</version>
```

Comment:
https://github.com/open-telemetry/opentelemetry-java-instrumentation/tree/main/instrumentation/spring/spring-boot-autoconfigure
uses `OPENTELEMETRY_VERSION` as version placeholders. I didn't check what other library instrumentations use, but it would be nice if we could follow the same pattern in all of them.